builtin/mainmenu/tab_local.lua

```diff
@@ -269,6 +269,12 @@ local function main_button_handler(this, fields, name, tabdata)
 	end
 
 	if fields["play"] ~= nil or world_doubleclick or fields["key_enter"] then
+		local enter_key_duration = core.get_us_time() - this.dlg_create_world_closed_at
```

Comment:
`dlg_create_world_closed_at` should be set to zero as soon as the tab is created. Otherwise it might be `nil` at this point (you can reproduce this by opening the game and trying to enter a world without opening the world creation dialog.)

Reply:
I added simple initialisation at the start of the `main_button_handler` function.